exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_data.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "opentelemetry/sdk/common/circular_buffer.h"
+#include "opentelemetry/sdk/trace/recordable.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+#include "vector"
```

Comment:
For header file from standard library, prefer including it via `#include <vector>`?